master/buildbot/test/unit/test_stats_service.py

```diff
@@ -154,9 +164,9 @@ def test_influx_service_not_inited(self):
         svc.thd_postStatsValue("test", "test", "test")
         self.assertLogged("Service.*not initialized")
 
-    def test_storage_backend_base(self):
-        # Just initlize and run thd_postStatsValue
-        svc = StatsStorageBase()
+    def test_storage_backend_base_failure_on_init(self):
+        svc = DummyStatsStorageBase()
```

Comment:
there are abstract method left. I use quite heavily abc and pylint only complains for the base class instanciation. Derived class should not complain if all abstractmethod are overriden in the inherited class